.circleci/config.yml

```diff
@@ -103,11 +103,53 @@ jobs:
       - run: lein uberjar
       - store_artifacts:
           path: target/uberjar/rems.jar
+      - persist_to_workspace:
+          root: target/uberjar
+          paths:
+            - rems.jar
+
+  rahti:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
```

Comment:
since we're not running any lein tasks I think this cache is unneeded
caching docker layers might make sense though

Reply:
Removed the cache. Will investigate docker layer caching